socket.io-client/socket.io-client.d.ts

```diff
@@ -1,46 +1,656 @@
-// Type definitions for socket.io-client 1.2.0
+// Type definitions for socket.io-client 1.3.5
 // Project: http://socket.io/
-// Definitions by: PROGRE <https://github.com/progre/>
+// Definitions by: divillysausages <https://github.com/divillysausages/>
```

Comment:
OK, sorry, I thought the author was updated with each revision. Want me to change it or leave it for @progre when he reviews the PR?